app/services/highlights_info.rb

```diff
@@ -30,7 +30,7 @@ def avg_highlights_per_user
           ( SELECT
               COUNT(*) AS highlights_count
             FROM highlights
```

Comment:
thought of that too...but also thought this could be more official...here...we're really counting the highlights.  postgres will do this internally, i would expect it to be efficient.   also..perhaps...in some situation we decide to reset the num_highlights/note chars...this would remain as the authoritative count.

Reply:
will performance test this as part of https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/625